extensions/ql-vscode/supported_cli_versions.json

```diff
@@ -1,5 +1,5 @@
 [
-  "v2.13.0",
+  "v2.13.1",
   "v2.12.7",
   "v2.11.6",
   "v2.7.6",
```

Comment:
Why are we still testing against 2.7.6? Shouldn't we be testing against 2.10.x instead?

Reply:
Ugh...ignore me. I didn't see that this file has 10 lines. I only saw the first 5.

Regardless, we only need to support the last 4 minor versions of the CLI.